boards/hifive1/src/main.rs

```diff
@@ -187,10 +187,6 @@ pub unsafe fn main() {
         &peripherals.e310x.gpio_port[17],
         &peripherals.e310x.gpio_port[16],
     );
-    peripherals.e310x.uart1.initialize_gpio_pins(
-        &peripherals.e310x.gpio_port[18],
-        &peripherals.e310x.gpio_port[23],
-    );
```

Comment:
Why remove this?

Reply:
Because when I placed it there, the clearing of pending interrupts didn't work. After the [patch](https://github.com/tock/tock/commit/2ce997d5680b9cdcd47c2ec70d25901132df2a40) there was no need for it.